README.md

```diff
@@ -103,6 +103,17 @@ const [styles] = createStyles({
 
 const myHeader = document.createElement('header');
 myHeader.classList.add(styles.header); // Will have a generated CSS classname in the format of '.header<unique_identifier>' ex .headerumdoaudnaoqwu
+
+// if you want Simplestyle to always start with the same seed for generating the random suffix applied to the generated CSS class names
```

Comment:
How about this as a comment?

// if you want Simplestyle to always  generate the same CSS class names, you can set
// your own initial seed. Assuming your modules are imported in the same order and
// execute their calls to createStyles() in the same order, the library will reliably generate
// the same classNames across successive calls.

Reply:
That reads much better 😄 

I'll update and push. Since this is a new API, I'm actually confident about pushing to prod. I'll ping you directly when I've published